src/openloco/localisation/unicode.cpp

```diff
@@ -0,0 +1,36 @@
+#include "unicode.h"
+#include <cstdint>
+
+namespace openloco::localisation
```

Comment:
`openloco::localisation::unicode`?

Reply:
Hmm, considering there are two types defined in the file (`utf8_t` and `utf32_t`), I'd rather keep them in the localisation namespace.